### DISCUSSION TITLE
Shortcut to open app not working?

### DISCUSSION THREAD

**Discussion by Antonio-Laguna** [johnlindquist/kit#130] [Q&A]
Hey!

I've been reading all about Script Kit and it's looking great. However the shortcuts to open them are not working for me. `cmd + ;` seems also to be the default shortcut to open settings in mac apps so how's that supposed to work?

Thanks for Script Kit!

**Comment by johnlindquist** [comment]
Hmm, "Open Settings" in mac apps is usually cmd+, for me. I'm curious, is your Mac set to a language other than English?

I'll prioritize allowing to custize the main shortcut.

For now, you should be able to click on the tray icon to launch Script Kit.

**Reply by Antonio-Laguna** [reply]
My Mac language is set to English yet my keyboard is Spanish which I think will be the differentiation here.

**Comment by filiptammergard** [comment]
Yes, this is the case for me as well. I have a Swedish keyboard that looks like this:

![image](https://user-images.githubusercontent.com/44197016/114382667-ea167880-9b8c-11eb-90bb-3dc6d1c95012.png)

Which means `;` and `,` is the same button, so there's a conflict with the open settings shortcut. Would be great to be able to customize the main shortcut!

**Comment by johnlindquist** [comment]
Here's an easy solution:

[Add "main-shortcut" to Script Kit](https://scriptkit.app/api/new?name=main-shortcut&url=https://gist.githubusercontent.com/johnlindquist/e5e4fd9fba697e57fcf6efd4b0d8ec64/raw/c92fff0cf1543e44a63815c61ca082c2f7189b15/main-shortcut.js)

```js
// Shortcut: cmd opt shift f
await main("index")
```

After you add that script, you can adjust the shortcut by playing with the `//Shortcut: ` comment

**Reply by Alarid** [reply]
This does the trick for me, just wondering if there is a way to hide this script from the menu ?

**Reply by johnlindquist** [reply]
Currently, no, but I'll definitely add that!

Probably a comment like

//Exclude

**Reply by Alarid** [reply]
That would be awesome, looking forward to that
